src/components/ServiceDetails/AboutService/ServiceDemo/Purchase/ExpiredSession/MetamaskFlow.js

```diff
@@ -189,7 +192,7 @@ class MetamaskFlow extends Component {
             <span className={classes.channelSelectionTitle}>Recommended</span>
             <ChannelSelectionBox
               title="Channel Balance"
-              description={`You have ${this.parseChannelBalFromPaymentCard()} AGI in you channel. This can be used for running demos across all the services from this vendor.`}
+              description={`You have 1 AGI in you channel. This can be used for running demos across all the services from this vendor.`}
```

Comment:
Do you want to dynamically show this?

Reply:
This is handled in https://github.com/singnet/refactored-dapp/pull/189